src/cli/Edit.h

```diff
@@ -24,8 +24,8 @@ class Edit : public DatabaseCommand
 {
 public:
     Edit();
-    ~Edit();
-    int executeWithDatabase(QSharedPointer<Database> db, QSharedPointer<QCommandLineParser> parser);
+    ~Edit() override = default;
```

Comment:
@sjamesr is there a reason why you decided to remove the destructor declaration in some commands, and declared it `override = default` in others?

Reply:
Oh, that was just an oversight, should be fixed now. Thanks!